### PR TITLE
feat(sql): Add sql language support

### PR DESCRIPTION
**Pull request by its-sushant**
pull request for #37

### CODE REVIEW

**Review by Kaushl2208 — CHANGES_REQUESTED** [review]
Hey, Can you add the test data file as a raw github content link. On which your test will run on.

**Comment by Kaushl2208** [comment]
Hey @its-sushant , You have few whitespaces in sql and main file! 
Try and remove it

**Comment by its-sushant** [comment]
> Hey @its-sushant , You have few whitespaces in sql and main file! Try and remove it

Ok, I will try to remove it..

File: source.txt [hunk 1]
```diff
@@ -0,0 +1,282 @@
+import { ThriftTest } from "./gen-js/ThriftTest_types";
```
Comment (GMishx) [inline source.txt:1]: Please remove invalid files from the commit.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: source.txt (hunk 1 above)
Comment (its-sushant) [reply source.txt:1]: Ok i will remove that

**Review by Kaushl2208 — APPROVED** [review]
Changes Looks good! Needs test.

**Comment by Kaushl2208** [comment]
Hey @its-sushant , Thankyou for your contributions. I have tested it

CC: @GMishx

**Pull request merged**